tests/countries/test_india.py

```
@@ -180,6 +180,8 @@ def _assertHinduHolidayHelper(  # noqa: N802
         *,
         category_optional: bool = False,
         subdivs: set | None = None,
+        hindu_range: Iterable[int] | None = None,
```

Ruff (0.15.12)

[error] Undefined name `Iterable`

(F821)